.changelogs/issue_2365.yml

```diff
@@ -0,0 +1,7 @@
+significance: patch
+type: changed
+comment: Added filter to change the text to be displayed when the student is not
+  enrolled in any courses.
+links:
+  - "#2396"
+entry: Added Filter for No Course Enrollement text.
```

Comment:
Should be updated with are reference to the memberships filter too.
Or you can create two different changelog entries.
I would also add a reference to the hook name in the `entry` field. E.g.

> Added `lifterlms_dashboard_courses_not_enrolled_text` filter hook to allow altering the message displaying on the dashboard when the current user is not enrolled in any courses.

(similar thing for the memberships)

Reply:
done.

Reply:
yeah the changelog thing is normal